src/nunit-gui/Model/ITestModel.cs

```diff
@@ -73,6 +73,10 @@ public interface ITestModel : IServiceLocator, System.IDisposable
 
         #region Properties
 
+        CommandLineOptions Options { get; }
```

Comment:
Maybe this should be called CommandLineOptions, CommandOptions, RunOptions or something else more specific.